tests/resources/crlf.git/COMMIT_EDITMSG

```diff
@@ -1 +1 @@
-Add .gitattributes
+subdir with no translation
```

Comment:
You might consider dropping some of the files in "crlf.git", like "description", "COMMIT_EDITMSG", "info/exclude".

Reply:
Fixed.